### PR TITLE
Cap reorg doubling

### PR DESCRIPTION
**Pull request by connorwstein**
To port as a hotfix on 10.8

### CODE REVIEW

File: core/services/run_manager.go [hunk 1]
```diff
@@ -204,12 +204,18 @@ SELECT coalesce(max(task_runs.minimum_confirmations), 0) FROM job_runs
 	if maxConfs != 0 {
 		for i := range run.TaskRuns {
 			if run.TaskRuns[i].TaskSpec.Type == adapters.TaskTypeRandom && run.TaskRuns[i].MinRequiredIncomingConfirmations.Valid {
+				// Sanity cap must be less than 256 otherwise the blockhash will not be available directly
+				// We give it 56 blocks as a buffer to still be able to fulfill within that bound.
+				newConfs := maxConfs * 2
+				if maxConfs*2 > 200 {
```
Comment (PiotrTrzpil) [inline core/services/run_manager.go:210]: minor but you could do `if newConfs > 200 {`

**Review by PiotrTrzpil — DISMISSED** [review]
looks good to me!

File: core/services/run_manager.go (hunk 1 above)
Comment (samsondav) [reply core/services/run_manager.go:210]: Why is it times by two twice... so maxConfs must be less than 50?

Comment (connorwstein) [reply core/services/run_manager.go:210]: no just multiplies once - updated to make it more clear with piotrs suggestion

**Review by PiotrTrzpil — APPROVED** [review]

**Pull request merged**